modules/criteoBidAdapter.js

```diff
@@ -0,0 +1,277 @@
+import { loadScript } from 'src/adloader';
+import { registerBidder } from 'src/adapters/bidderFactory';
+import { EVENTS } from 'src/constants';
+import { parse } from 'src/url';
+import * as utils from 'src/utils';
+
+var events = require('src/events');
+
+const ADAPTER_VERSION = 4;
+const BIDDER_CODE = 'criteo';
+const CDB_ENDPOINT = '//bidder.criteo.com/cdb';
+const INTEGRATION_MODES = {
+  'amp': 1,
+};
+const PROFILE_ID = 207;
+const PUBLISHER_TAG_URL = '//static.criteo.net/js/ld/publishertag.prebid.js';
```

Comment:
Can you put a comment here point to https://github.com/Prebid-org/prebid-js-external-js-criteo/blob/master/dist/prod.js as source?

Reply:
Done.